src/main/java/edu/kit/hci/soli/config/WebSecurityConfig.java

```diff
@@ -23,7 +23,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .authorizeHttpRequests(requests ->
                         requests
                                 .requestMatchers("/", "/api/events").permitAll()
-                                .requestMatchers("/*").authenticated()
+                                .requestMatchers("/**").authenticated()
```

Comment:
The old code only permitted access to paths with a depth of one. This fixes that